fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/digitalcash/TransactionObjectBuilder.java

```diff
@@ -0,0 +1,72 @@
+package com.github.dedis.popstellar.model.objects.digitalcash;
+
+import com.github.dedis.popstellar.model.objects.Channel;
+import com.github.dedis.popstellar.model.objects.InputObject;
+import com.github.dedis.popstellar.model.objects.OutputObject;
+
+import java.util.List;
+
+public class TransactionObjectBuilder {
+
+  private Channel channel;
+
+  // version
+  private int version;
```

Comment:
Should these be final as well like for the `TransactionObject` ?

Reply:
No, we want to set it freely